site/cypress/e2e/Cypress-Gradeable/late_submission_warning_messages.spec.js

```diff
@@ -118,66 +130,95 @@ const SubmitAndCheckMessage = (gradeable_type, upload_file1, invalid_late_day, v
-    cy.logout();
+describe('Checks whether daylight savings warning message should be appearing given varying amounts of late days.', () => {
+    it('should create non-team gradeable for testing daylight savings', () => {
+        makeNonTeamGradeable(daylight_gradeable);
+
+        const date = new Date().toISOString();
+        // change all the due dates to today
+        changeAllDates(daylight_gradeable, date);
+    });
+
+    it('test if daylight savings banner should appear for different amount of late days', () => {
+        cy.login('instructor');
+        checkDaylightBanner(1000, 'exist');
+        checkDaylightBanner(0, 'not.exist');
+        calculateAndCheckDaylightBanner(1);
+        calculateAndCheckDaylightBanner(99);
+        calculateAndCheckDaylightBanner(100);
+        calculateAndCheckDaylightBanner(101);
+    });
+
+    it('test that daylight savings banner should not appear when we are past the due date', () => {
+        cy.login('instructor');
+
+        // first day in 2001
+        const date = new Date('2001').toISOString();
+        console.log(date);
+        changeAllDates(daylight_gradeable, date);
+        checkDaylightBanner(0, 'not.exist');
+        checkDaylightBanner(200, 'not.exist');
+        checkDaylightBanner(1000, 'not.exist');
+    });
 });
 
 describe('Test warning messages for non team gradeable', () => {
     it('should create non-team gradeable for testing', () => {
-        cy.login('instructor');
-        cy.visit(['sample', 'gradeable']);
-        // Enter gradeable info
-        cy.get('#g_title').type(gradeable);
-        cy.get('#g_id').type(gradeable);
-        cy.get('#radio_ef_student_upload').check().click();
-        // Create Gradeable
-        cy.get('#create-gradeable-btn').click();
-
+        makeNonTeamGradeable(gradeable);
         // Date page, input 2 old dates for opening dates (Ta and students)
         cy.get('#page_5_nav').click();
-        cy.get('#date_ta_view')
-            .clear()
-            .type('1992-06-15')
-            .type('{enter}');
-        cy.get('#save_status', { timeout: 20000 }).should('have.text', 'All Changes Saved');
-        cy.get('#date_submit')
-            .clear()
-            .type('2004-12-18')
-            .type('{enter}');
-        cy.get('#save_status', { timeout: 20000 }).should('have.text', 'All Changes Saved');
+        cy.get('[data-testid=ta-view-start-date]').clear();
+        cy.get('[data-testid=ta-view-start-date]').type('1992-06-15');
+        cy.get('[data-testid=ta-view-start-date]').type('{enter}');
+        cy.get('[data-testid=save-status]', { timeout: 20000 }).should('have.text', 'All Changes Saved');
+        cy.get('[data-testid=submission-open-date]').clear();
+        cy.get('[data-testid=submission-open-date]').type('2004-12-18');
+        cy.get('[data-testid=submission-open-date]').type('{enter}');
+        cy.get('[data-testid=save-status]', { timeout: 20000 }).should('have.text', 'All Changes Saved');
     });
```

Comment:
Same comment above about this could (and should) probably be in a before block, but since this part of the PR is just changing to testids and linting, this one could be done in a future PR